lazy-imports-import.html

```diff
@@ -19,35 +21,28 @@
      *
      * `importHref` is only called once per globally-unique URL.
      *
-     * @param {HTMLElement} callingElement The element that is used for
-     *                                     resolving imports and fallback
-     *                                     for the dom-module if no
-     *                                     fromElment is passed.
-     * @param {string=} group The group attribute to filter on.
-     * @param {HTMLElement=} fromElement The element to search for lazy
+     * @param {HTMLElement} fromElement The element to search for lazy
      *                                   imports within. If no element is
      *                                   provided, this behavior will find a
-     *                                   `dom-module` with an id matching
+     *                                   `dom-module` with an `id` matching
```

Comment:
An element has to always be provided now, yes? Also, it might make sense to allow a string id to be passed here too.